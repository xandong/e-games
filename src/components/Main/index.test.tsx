import { render, screen } from "@testing-library/react"
import Main from "."

describe("<Main />", () => {
  it("should be render element", () => {
    render(<Main />)

    expect(
      screen.getByRole("heading", {
        name: /main component/i
      })
    )
  })
})
