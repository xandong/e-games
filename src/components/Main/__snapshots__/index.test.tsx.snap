// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should be render element 1`] = `
<main>
  <h1>
    Main Component
  </h1>
</main>
`;

exports[`<Main /> should render the element 1`] = `
<main>
  <h1>
    Main Component
  </h1>
</main>
`;
